style.css

```diff
@@ -36,7 +37,7 @@ body {
 
 .toolbar-section {
   background-repeat: no-repeat;
-  background-image: url("/Images/header-shapes\ mobile.png"), url("/Images/header-shapes\ mobile@2x.png");
+  background-image: url("/Images/header-shapes\ mobile@2x.png"), url("/Images/header-background-mobile.png");
   border: 1 px solid #fff;
 }
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you correctly check the **padding-bottom** of the **Home section**and its **border-radius** at **left-bottom** to make the design stick to the template.